index.html

```diff
@@ -48,7 +48,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
```suggestion
<h2 hidden>Game over! Want to play again?! Just click refresh. 🧑‍🚀!</h2>
```
Let's make it a bit more friendly. 🤓